Wikipedia/UI-V5/WMFHomeNearbyCell.m

```diff
@@ -0,0 +1,162 @@
+
+#import "WMFHomeNearbyCell.h"
+
+#import "Wikipedia-Swift.h"
+#import "PromiseKit.h"
+
+@interface WMFHomeNearbyCell ()
+
+@property (strong, nonatomic) IBOutlet UIImageView *imageView;
```

Comment:
@bgerstle   Xcode's default/guidance for iBOutlets is strong for iOS projects under ARC. Some older documentation may say weak… but that was before iOS 6.

This Stackoverflow answer points to the new guidance as given at WWDC, along with Twitter conversations with Apple Engineers confirming the change and outdated docs: 
http://stackoverflow.com/a/31395938/48311

The relevant WWDC session: http://asciiwwdc.com/2015/sessions/407

Basically, Apple recommends that you use strong unless you have memory semantic issues that would cause you to do otherwise.

cc @montehurd